### ISSUE TITLE
Dnu Install ArgumentNullException

### ISSUE THREAD

**Issue by raynjamin** [aspnet/Home#685]
Hello Folks,

Forgive me if this is a duplicate issue or if this is a simple fix- I have tried to do my due diligence in researching this issue but I haven't seen anyone with the same problem as of yet.

I'm just getting started with the dnvm/dnu/dnx/vs code shebang. I'm trying to do something quite simple-- install a package. Each time I attempt to install said package, I get an "ArgumentNullException."

```
λ dnu install EntityFramework
Microsoft .NET Development Utility CLR-x86-1.0.0-beta5-12021

  GET https://www.nuget.org/api/v2/FindPackagesById()?id='EntityFramework'.
  OK https://www.nuget.org/api/v2/FindPackagesById()?id='EntityFramework' 815ms
EntityFramework.7.0.0-beta4 was added to project.json.
System.ArgumentNullException: Value cannot be null.
Parameter name: path
   at System.IO.Path.GetFullPathInternal(String path)
   at System.IO.Path.GetFullPath(String path)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.<DistinctIterator>d__1`1.MoveNext()
   at Microsoft.Framework.PackageManager.RestoreCommand.<Execute>d__75.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.Framework.PackageManager.InstallCommand.<ExecuteCommand>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Framework.PackageManager.InstallConsoleCommand.<>c__DisplayClass0_0.<<Register>b__1>d.MoveNext()
```

Anyone have any ideas?

Thanks,
Ben

**Comment by dougbu** [comment]
Make sure you're using a Beta 4 runtime. Have a look at the output of `dnvm list`; that should show `1.0.0-beta4` as selected and (likely) with the `default` alias. `dnvm upgrade` (no `-u`) if necessary.

If everything is fine, try another package to see if the issue is specific to `EntityFramework`. For example you might try `Microsoft.Framework.Logging.Console` though I'm not sure that's the right name  in Beta 4.

**Comment by raynjamin** [comment]
Thank you! This problem affected all packages for me, but it was solved by reverting back to 1.0.0-beta4. Dnvm was set to use 1.0.0-beta5-12021 by default.

:+1:
